src/OnlineSales/Entities/Account.cs

```diff
@@ -0,0 +1,43 @@
+﻿// <copyright file="Account.cs" company="WavePoint Co. Ltd.">
+// Licensed under the MIT license. See LICENSE file in the samples root for full license information.
+// </copyright>
+
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text.Json.Serialization;
+using OnlineSales.DataAnnotations;
+
+namespace OnlineSales.Entities;
+
+[Table("account")]
+[SupportsChangeLog]
+public class Account : BaseEntity
+{
+    public string Name { get; set; } = string.Empty;
```

Comment:
"Name" should be made unique by adding [Index(nameof(Name), IsUnique = true)] to Account class.